src/lib/rest/rest.h

```diff
@@ -133,4 +139,21 @@ extern void firstServicePath(const char* servicePath, char* servicePath0, int se
 */
 extern bool isOriginAllowedForCORS(const std::string& requestOrigin);
 
+
+
+/* ****************************************************************************
+*
+* serviceVectorsSet - only for unit tests
```

Comment:
Repeated declaration? (in seems is also at RestService.h)

Reply:
Fixed in a265c79